Framework/Algorithms/test/StripVanadiumPeaks2Test.h

```diff
@@ -92,8 +92,9 @@ class StripVanadiumPeaks2Test : public CxxTest::TestSuite {
     const std::vector<double> PEAK_X{1.2356, 1.5133, 2.1401}; // peak positions in d-spacing
     const std::vector<double> PEAK_Y{28.5, 37.8, 12.3};       // low precision results
                                                               // of stripped peaks
-    // something is wrong with the second peak on windows
-    const std::vector<double> TOL{.1, 10., .1};
+    // something is wrong with the second peak on windows, something seems to be wrong with the peaks on Ubuntu in
```

Comment:
I'm not terribly comfortable with this as a fix. It smells of something we don't understand. This test passes with Conda on CentOS?
If we leave the tolerances like this it doesn't feel like a very effective test. Can we save the test data out and see what it looks like? I wonder if this is another case of what @StephenSmith25 found with the SANS data in that it's an unstable fit so any of these answers is technically correct but it does not make for a reliable test.

Reply:
I spoke to Richard about this, and he seemed to think it was to do with the peaks, he actually guessed which values would be wrong before I told him. My basic understanding is that the values are technically correct but not good for reliable testing.

Reply:
Can we find some better values?

Reply:
The tolerance needs to allow for between 27.1 and 28.5 so I could use 1.5 instead of 10.? I copied what was already in the file for "dodgy" data.

Reply:
I guess I'm still not clear on why the values change...
I think atleast slimming them to the minimum necessary would be good and possible putting in an OS check to make that clearer?